_sass/custom/custom.scss

```diff
@@ -54,6 +54,12 @@ code {
 
 .site-nav {
   padding-top: 1rem;
+
+  display: static;
+  position: sticky;
```

Comment:
`static` is not a valid value for `display`. I think you meant to do `position: static` as a fallback for `position: sticky` which according to [caniuse](https://caniuse.com/css-sticky) is already supported by all the browser versions that matter.

Reply:
Thank you. Removed.